### PR TITLE
Better indication of service class and possible implementations

### PR DESCRIPTION
**Pull request by gastaldi**
#### Short description of what this resolves:
This makes the error message clearer

#### Changes proposed in this pull request:

- Error message

### CODE REVIEW

File: core/impl-base/src/main/java/org/jboss/arquillian/core/impl/loadable/ServiceRegistryLoader.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public ServiceRegistryLoader(Injector injector, ServiceRegistry registry) {
         }
         if (all.size() > 1) {
             throw new IllegalStateException(
-                "Multiple service implementations found for " + serviceClass + ". " + toClassString(all));
+                "Multiple service implementations found for " + serviceClass + ": " + toClassString(all));
```
Comment (lordofthejars) [inline core/impl-base/src/main/java/org/jboss/arquillian/core/impl/loadable/ServiceRegistryLoader.java]: Would it make it more clear add some white spaces as well? `" : "`

Comment (gastaldi) [reply core/impl-base/src/main/java/org/jboss/arquillian/core/impl/loadable/ServiceRegistryLoader.java]: Not sure, that wouldn't be grammatically correct I think

Comment (lordofthejars) [reply core/impl-base/src/main/java/org/jboss/arquillian/core/impl/loadable/ServiceRegistryLoader.java]: Ok :) it was just a comment :)

Comment (bartoszmajsak) [reply core/impl-base/src/main/java/org/jboss/arquillian/core/impl/loadable/ServiceRegistryLoader.java]: Not sure if grammar has anything to do here :)

**Comment by bartoszmajsak** [comment]
Thanks @gastaldi.

**Pull request merged**